integreat_cms/cms/templates/hix_widget.html

```diff
@@ -64,7 +65,61 @@
                 <i icon-name="refresh-ccw"></i>
                 {% translate "Update" %}
             </button>
-            <label class="secondary">
+            <div id="hix-feedback" class="hidden">
+                <label class="secondary">
+                    {% translate "For review:" %}
+                </label>
+                <div hix-feedback-category="hix-count-nested-sentences"
+                     class="help-text mt-2 hidden">
+                    <i icon-name="bar-chart-big" class="size-6 pr-1"></i>
+                    {% translate "The text contains" %}
+                    <span class="font-bold"></span>
+                    <span class="font-bold">{% translate "nested sentences" %}</span>
```

Comment:
This seems unnecessary. I believe it would be better to look for an alternative solution if this is being used for layout.

Reply:
These spans get populated from javascript: https://github.com/digitalfabrik/integreat-cms/pull/2761/files#diff-3334c5517cdff73b29d44ccb02b3d4b1b384c84ccaa19345543ca485bcaa5a30R105

Reply:
That makes sense. Thanks for the explanation @david-venhoff!